src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -360,6 +380,9 @@ def __new_installation_summary_json(self, installation_packages_json):
 
     def set_patch_metadata_for_healthstore_substatus_json(self, status=Constants.STATUS_SUCCESS, code=0, patch_version=Constants.PATCH_VERSION_UNKNOWN, report_to_healthstore=False, wait_after_update=False):
         """ Prepare the healthstore substatus json including message containing summary to be sent to healthstore """
+        if self.execution_config.exec_auto_assess_only:  # Previously included: status == Constants.STATUS_TRANSITIONING
```

Comment:
NIT: There's a high possibility we will forget to remove this comment later on. My suggestion would be to remove it from here and make a note in the PR description and keep an eye out for any issues after this change goes in. 

Also, previous doesn't seem right, because technically this is the first edition for it so there was nothing before

Reply:
Good idea, will include in the PR description.